Riot/Modules/Room/Views/Title/RoomTitleView.m

```diff
@@ -109,6 +118,22 @@ - (void)refreshDisplay
     }
     else if (self.mxRoom)
     {
+        if (!mxDirectUserPresenceObserver && self.mxRoom.isDirect)
+        {
+            // Observe contact presence change
+            mxDirectUserPresenceObserver = [[NSNotificationCenter defaultCenter] addObserverForName:kMXKContactManagerMatrixUserPresenceChangeNotification object:nil queue:[NSOperationQueue mainQueue] usingBlock:^(NSNotification *notif) {
+                
+                NSString* directUserId = self.mxRoom.directUserId;
```

Comment:
Would it be worth having a `MXWeakify(self)`/`MXStrongifyAndReturnIfNil(self)` for this block?

Reply:
Seems reasonable, I added it on all occurrences of this observer